style.css

```diff
@@ -43,10 +43,41 @@
 }
 
 .list_item_active a{
-    color: #e8491d;
+    color:#C1EDFE;
 
 }
 
 .header_nav_list a:hover{
     color: rgb(241, 245, 8);
-}
\ No newline at end of file
+    font-size: 25px;
+
+}
+
+
+ /*Search Style */
+
+.search{
+    justify-content: center; 
+    width:70%;
+    height: 70%;
+    border: 3px solid #C1EDFE;
+    border-radius: 25px;
+    padding: 5px;
+    margin-top: 5px;
+    margin-right: 5px;
+    font-size: 5px;
+}
```

Comment:
remove the outline border when you click on the button! 

![Screenshot from 2021-03-03 22-34-47](https://user-images.githubusercontent.com/47992412/109868912-ea0f8880-7c70-11eb-8632-fca9dcb92ac3.png)